bundles/net.tourbook/src/net/tourbook/ui/views/TourBlogView.java

```diff
@@ -99,35 +101,36 @@
+   public static final String  ID                                              = "net.tourbook.ui.views.TourBlogView";         //$NON-NLS-1$
 
    private static final String NL                                              = UI.NEW_LINE1;
 
-   private static final String SPACER                                          = "<div>&nbsp;</div>";                       //$NON-NLS-1$
+   private static final String SPACER                                          = "<div>&nbsp;</div>";                          //$NON-NLS-1$
 
-   private static final String TOUR_BLOG_CSS                                   = "/tourbook/resources/tour-blog.css";       //$NON-NLS-1$
+   private static final String TOUR_BLOG_CSS                                   = "/tourbook/resources/tour-blog.css";          //$NON-NLS-1$
 
-   static final String         STATE_IS_DRAW_MARKER_WITH_DEFAULT_COLOR         = "STATE_IS_DRAW_MARKER_WITH_DEFAULT_COLOR"; //$NON-NLS-1$
+   static final String         STATE_IS_DRAW_MARKER_WITH_DEFAULT_COLOR         = "STATE_IS_DRAW_MARKER_WITH_DEFAULT_COLOR";    //$NON-NLS-1$
    static final boolean        STATE_IS_DRAW_MARKER_WITH_DEFAULT_COLOR_DEFAULT = false;
-   static final String         STATE_IS_SHOW_HIDDEN_MARKER                     = "STATE_IS_SHOW_HIDDEN_MARKER";             //$NON-NLS-1$
+   static final String         STATE_IS_SHOW_HIDDEN_MARKER                     = "STATE_IS_SHOW_HIDDEN_MARKER";                //$NON-NLS-1$
    static final boolean        STATE_IS_SHOW_HIDDEN_MARKER_DEFAULT             = true;
-   static final String         STATE_IS_SHOW_TOUR_TAGS                         = "STATE_IS_SHOW_TOUR_TAGS";                 //$NON-NLS-1$
+   static final String         STATE_IS_SHOW_TOUR_TAGS                         = "STATE_IS_SHOW_TOUR_TAGS";                    //$NON-NLS-1$
    static final boolean        STATE_IS_SHOW_TOUR_TAGS_DEFAULT                 = true;
 
-   private static final String EXTERNAL_LINK_URL                               = "http";                                    //$NON-NLS-1$
-   private static final String HREF_TOKEN                                      = "#";                                       //$NON-NLS-1$
-   private static final String PAGE_ABOUT_BLANK                                = "about:blank";                             //$NON-NLS-1$
+   private static final String EXTERNAL_LINK_URL                               = "http";                                       //$NON-NLS-1$
+   private static final String HREF_TOKEN                                      = "#";                                          //$NON-NLS-1$
+   private static final String PAGE_ABOUT_BLANK                                = "about:blank";                                //$NON-NLS-1$
+   private static final String VALUE_UNIT_KCALORIES                            = net.tourbook.ui.Messages.Value_Unit_KCalories;
```

Comment:
Message texts from other messages.properties should always be retrieved from `net.tourbook.OtherMessages`

Reply:
Fixed